src/delta.rs

```diff
@@ -491,7 +491,10 @@ fn handle_hunk_header_line(
         painter
             .line_numbers_data
             .initialize_hunk(line_numbers, plus_file.to_string());
-    } else if config.line_numbers_show_first_line_number && !config.hunk_header_style.is_raw {
+    } else if config.line_numbers_show_first_line_number
+        && !config.hunk_header_style.is_raw
+        && !config.color_only
+    {
```

Comment:
If you're able to add a comment explaining this boolean logic (here and in other places) that would be really helpful. I'm finding that when I haven't worked on the code for a while that these boolean conditions are very hard to understand!

Reply:
I see. I'm going to add.

Reply:
@dandavison 
added comment here.
ed9f3c9